ga4gh/cli.py

```diff
@@ -357,10 +361,18 @@ def __init__(self, args):
         super(SearchVariantsRunner, self).__init__(args)
         request = RequestFactory(args).createSearchVariantsRequest()
         # if no variantSets have been specified, send a request to
-        # the server to grab all variantSets and then continue
+        # the server to grab all variantSets in the first dataset
```

Comment:
Why the first data set here? Shouldn't we do this for all datasets for consistency?